subsys/bluetooth/host/adv.c

```diff
@@ -1153,6 +1153,8 @@ static int le_ext_adv_param_set(struct bt_le_ext_adv *adv,
 	cp->filter_policy = get_filter_policy(param->options);
 	cp->tx_power = BT_HCI_LE_ADV_TX_POWER_NO_PREF;
 
+	adv->options = param->options;
```

Comment:
I didn't `options` in `struct bt_le_ext_adv ` did I miss something?

Reply:
nvm I got confused because it was at the end of the diff since it's a private include

Reply:
No problem. I had the opposite problem before, where I added the field but then never set any value.

Reply:
No problem. I had the opposite problem before, where I added the field but then never set any value.